pkgs/by-name/ep/epy/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, python3Packages, fetchPypi }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "epy";
+  version = "2023.6.11";
```

```suggestion
  version = "2023.6.11";
  pyproject = true;
```

this likely also requires `python3Packages.setuptools` in nativeBuildInputs

---

Thank you for reviewing. Could you please clarify the suggestion of adding pyproject = true? 

I added the setuptools as build inputs to adhere with your checklist, and set format = "setuptools"; Could you please review again?

---

currently the default in `buildPythonApplication` is `format = "setuptools";`, which we're trying to move away from since the python ecosystem is moving towards PEP621. `pyproject = true;` is a simpler different way of writing `format = "pyproject";`, and will become the default once we remove `format = "setuptools"`.

---

Thank you for the clarification 🙏builds fine with python3Packages.poetry-core.